src/beamlime/executables/raw_detectors_fastapi.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2024 Scipp contributors (https://github.com/scipp)
+import asyncio
+
+import uvicorn
+from fastapi import FastAPI, WebSocket
+
+from beamlime.core.websocket_manager import WebSocketManager
+from beamlime.executables.show_detector import (
+    make_arg_parser,
+    make_factory,
+    run_show_detector,
+)
+
+app = FastAPI()
+factory = make_factory()
+manager = factory[WebSocketManager]
+
+
+@app.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
+    await manager.connect(websocket)
+    try:
+        while True:
+            _ = await websocket.receive_text()
```

Comment:
What is happening here? What data is the websocket trying to recieve?

Reply:
I think the whole bit is unused now, I was just doing some initial testing. I'll see if it can be removed.